src/write_http.c

```diff
@@ -468,9 +471,9 @@ static int wh_write_kairosdb(const data_set_t *ds,
     }
   }
 
-  status = format_kairosdb_value_list(cb->send_buffer, &cb->send_buffer_fill,
-                                      &cb->send_buffer_free, ds, vl,
-                                      cb->store_rates);
+  status = format_kairosdb_value_list(
+      cb->send_buffer, &cb->send_buffer_fill, &cb->send_buffer_free, ds, vl,
+      cb->store_rates, (char const *const *)http_attrs, http_attrs_num);
```

Comment:
If I change to `static const char **http_attrs` it trigger error on strarray_add() which take a char *** as first parameter : 

```
src/write_http.c: In function ‘wh_config_node’:
src/write_http.c:731:20: error: passing argument 1 of ‘strarray_add’ from incompatible pointer type [-Werror=incompatible-pointer-types]
       strarray_add(&http_attrs, &http_attrs_num, key);
                    ^
In file included from src/write_http.c:28:0:
./src/daemon/common.h:380:5: note: expected ‘char ***’ but argument is of type ‘const char ***’
 int strarray_add(char ***ret_array, size_t *ret_array_len, char const *str);
     ^~~~~~~~~~~~
src/write_http.c:732:20: error: passing argument 1 of ‘strarray_add’ from incompatible pointer type [-Werror=incompatible-pointer-types]
       strarray_add(&http_attrs, &http_attrs_num, val);
                    ^
In file included from src/write_http.c:28:0:
./src/daemon/common.h:380:5: note: expected ‘char ***’ but argument is of type ‘const char ***’
 int strarray_add(char ***ret_array, size_t *ret_array_len, char const *str);
     ^~~~~~~~~~~~
cc1: all warnings being treated as errors
```